src/main/java/com/codessquad/qna/repository/Question.java

```diff
@@ -28,8 +29,26 @@
     private String contents;
     private LocalDate createdAt = LocalDate.now();
 
+    public Question () {}
+    public Question(String title, String contents, User user) {
+        this.title = title;
+        this.contents = contents;
+        this.writer = user;
+    }
+
     public void update(Question question) {
         this.title = question.title;
         this.contents = question.contents;
     }
+
+    public boolean isCorrectFormat(Question question) {
+        boolean titleIsExist = ObjectUtils.isNotEmpty(question.getTitle());
+        boolean contentIsExist = ObjectUtils.isNotEmpty(question.getContents());
+        boolean writerIsExist = ObjectUtils.isNotEmpty(question.getWriter());
+        return titleIsExist && contentIsExist && writerIsExist;
+    }
+
+    public boolean isCorrectWriter(User user) {
+        return this.writer.getId().compareTo(user.getId()) == 0;
```

Comment:
id만 같은 경우, 같다고 표현하고 싶다면

Usre 객체 내부에 equals, hashCode를 재정의해서 아이디만 비교하게 한 뒤 아래와 같이 변경할수도 있습니다.

```suggestion
        return this.writer.equals(user);
```